sql/core/src/test/scala/org/apache/spark/sql/execution/python/TransformWithStateInPandasStateServerSuite.scala

```diff
@@ -574,6 +574,16 @@ class TransformWithStateInPandasStateServerSuite extends SparkFunSuite with Befo
     verify(arrowStreamWriter).finalizeCurrentArrowBatch()
   }
 
+  test("utils request - parse string schema") {
```

Comment:
Shall we also have the test for negative case?

Reply:
Since we use a bigger try-catch block under `run()` method, it's hard to test with. We could either have a specific try-catch block just for this call, or have a negative test on python side, which one do you prefer? cc @jingz-db

Reply:
I'm OK with having negative test on python side. I'm also OK with having follow up PR for this.